nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -64,7 +61,20 @@ public class NullAwaySerializationTest extends NullAwayTestsBase {
   private static final String SUGGEST_FIX_FILE_NAME = "fixes.tsv";
   private static final String SUGGEST_FIX_FILE_HEADER = SuggestedNullableFixInfo.header();
   private static final String ERROR_FILE_NAME = "errors.tsv";
-  private static final String ERROR_FILE_HEADER = ErrorInfo.header();
+  private static final String ERROR_FILE_HEADER =
+      String.join(
+          "\t",
+          "message_type",
+          "message",
+          "enc_class",
+          "enc_member",
+          "offset",
+          "target_kind",
+          "target_class",
+          "target_method",
+          "param",
+          "index",
+          "uri");
```

Comment:
Why not get this from `SerializationV2Adapter.getErrorsOutputFileHeader()`? Also, shouldn't we have both a test for V1 and a test for V2, based on which fix serialization config is passed? Make sure we really maintain support for both versions/formats.

Reply:
I think we discussed before it is better to see the actual expected value instead of calling APIs in tests, but changed it anyway to call `getErrorsOutputHeader()` as it is clearer. 
6c1263c96dbd73148534d62d6d552637d43f577f

Reply:
Added a test for serialization version 1. f7f8b32f31262f3bdb506aef5038876d082d6749 and b6df3e5ab4df194cd7d36010d19ba1240d88ca2c.

Currently all tests are running with serialization version `2`. With this added test, we specifically test the expected output conforms with version `1`. There might be better ways to set serialization version via `SerializationTestHelper` but I thought doing that might be out of scope of this PR and we do it in a follow up PR. Please let me know if you would like to have it in the same PR.